api/routes/index.js

```diff
@@ -1,4 +1,7 @@
 const express = require("express");
 const router = express.Router();
 
+const userRouter = require("./userRouter");
+router.use("/users", userRouter);
+
 module.exports = router;
```

Comment:
```suggestion
const express = require("express");

const userRouter = require("./userRouter");

const router = express.Router();

router.use("/users", userRouter);

module.exports = router;
```
모듈을 require하는 코드가 먼저 작성되는게 일반적인 관례입니다.